ntbs-service/Models/Enums/AlertType.cs

```diff
@@ -1,9 +1,11 @@
+using System.ComponentModel;
 using System.ComponentModel.DataAnnotations;
 
 namespace ntbs_service.Models.Enums 
 {
     public enum AlertType 
     {
+        [Display(Name = "Enhanced surveillance - MDR")]

```

Comment:
Are we preferring DisplayName here now?

Reply:
Forgot to mention this earlier when you asked - DisplayName doesn't work with enums when displaying it sadly so have to use Display(Name